src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  flex-direction: row;
+  padding-inline-start: 0;
+  list-style-type: none;
+  margin-bottom: 24px;
+  margin-right: 0;
+  margin-top: 16px;
+
+  &__star {
+    display: inline-block;
```

Comment:
Avoid using `display: inline-block` with flex items. Since the parent `.stars` is a flex container, you don't need to set `display` for the `.stars__star` elements, as they are automatically flex items.